src/compositions/question-form-8.tsx

```diff
@@ -0,0 +1,126 @@
+import { FC, useEffect, useState } from 'react';
+import { InputField } from '../elements/input-field';
+import { useRouter } from 'next/dist/client/router';
+import { Copy } from '../identity/copy';
+
+type Props = {
+  label?: string;
+  href: string;
+  answer: { [key: string]: string };
+  callback: React.Dispatch<React.SetStateAction<{ [key: string]: string }>>;
+};
+
+const validateInput = (answer: string) => {
+  if (parseFloat(answer) > 100) {
+    return '100 prozent ist das max';
+  }
+  if (parseFloat(answer) < 0) {
+    return 'Die Prozentangabe darf nicht negativ sein.';
+  }
+  return null;
+};
+export const Question8: FC<Props> = ({ label, href, callback, answer }) => {
+  const initialAnswer = '25%';
+  const router = useRouter();
+  const [totalAmount, setTotalAmount] = useState(0);
+
+  useEffect(() => {
+    if (answer === { Auto: '', ÖV: '', Fahrrad: '', Fuss: '' }) {
+      callback({ Auto: '25', ÖV: '25', Fahrrad: '25', Fuss: '25' });
+    } else {
+      callback(answer);
+    }
+    setTotalAmount(
+      (parseFloat(answer.Auto) || 0) +
```

Comment:
Are you sure you want to go with `float` and not `int`/`parseInt`? I doubt anyone's going to say "hey we travel by car in approx. 37.5712786235876% of all cases" :D